### PR TITLE
Fix passing on HTML/CSS options

### LINKED ISSUE #92: Fix passing on HTML/CSS options
CSS and HTML options are no longer passed on from `bankai start` and CSS options are no longer passed on to sheetify from bankai.

### PR DESCRIPTION
**Pull request by aknuds1**
Fixes #92

### CODE REVIEW

**Comment by aknuds1** [comment]
I wish there were still automated tests, so I wouldn't have to spend so much time hunting this down.

File: bin.js [hunk 1]
```diff
@@ -103,7 +104,11 @@ function main (argv) {
 }
 
 function start (entry, argv, done) {
-  const assets = bankai(entry)
+  const assets = bankai(entry, {
```
Comment (aknuds1) [inline bin.js]: Make sure that options are passed on to bankai.

File: index.js [hunk 2]
```diff
@@ -24,13 +24,14 @@ function Bankai (entry, opts) {
 
   const self = this
 
-  this.optimize = opts.optimize
   this.htmlDisabled = opts.html
-  this.cssDisabled = opts.css
   this.cssQueue = []
+  opts = xtend(opts, {
+    cssDisabled: opts.css === false
```
Comment (aknuds1) [inline index.js]: Move `cssDisabled` to opts, since `_javascript` doesn't have access to `this`.

File: index.js [hunk 3]
```diff
@@ -93,13 +94,15 @@ function _javascript (entry, opts, setCss) {
...
 
-  opts = xtend(base, opts || {})
+  const jsOpts = xtend(base, opts.js || {})
 
-  const b = (this.optimize)
-    ? browserify(opts)
-    : watchify(browserify(opts))
-  b.plugin(cssExtract, { out: createCssStream })
-  b.transform(sheetify)
+  const b = opts.optimize
```
Comment (aknuds1) [inline index.js]: Doesn't have access to `this` AFAICT.

File: index.js [hunk 4]
```diff
@@ -93,13 +94,15 @@ function _javascript (entry, opts, setCss) {
...
+    ? browserify(jsOpts)
+    : watchify(browserify(jsOpts))
+  if (!opts.cssDisabled) {
```
Comment (aknuds1) [inline index.js]: Not sure if this is the correct way to implement `cssDisabled`, but I don't have any reference to go by.

**Comment by bcomnes** [comment]
lgtm.

File: index.js (hunk 3 above)
Comment (bcomnes) [inline index.js]: It looks like its in scope, but then again, I was having some weird assignment issues over in https://github.com/yoshuawuyts/bankai/issues/87#issuecomment-261153648 which smelt of scope issues.

Comment (aknuds1) [inline index.js]: @bcomnes `this` is not in scope, I just checked :) It should not be in scope, since `_javascript` is not nested within the constructor.

**Comment by bcomnes** [comment]
@yoshuawuyts you cool to merge this?

**Comment by yoshuawuyts** [comment]
Closing in favor of #101

**Comment by yoshuawuyts** [comment]
Thanks heaps for the patch by the way! :tada:

**Pull request closed without merging**